packages/lib/SyncTargetJoplinServerSAML.ts

```diff
@@ -72,6 +72,41 @@ export default class SyncTargetJoplinServerSAML extends SyncTargetJoplinServer {
 		return false;
 	}
 
+	public static override async checkConfig(fileApi: FileApiOptions) {
+		try {
+			// Simulate a login request
+			const result = await fetch(`${fileApi.path()}/api/saml`);
+
+			if (result.status === 200) { // The server successfully responded, SAML is enabled
+				return {
+					ok: true,
+					errorMessage: '',
+				};
+			} else { // SAML is disabled or an error occurred
+				let message = result.statusText;
+
+				// Check if we got an error message
+				if (result.headers.get('Content-Type').includes('application/json')) {
+					const json = await result.json();
+
+					if (json.error) {
+						message = json.error;
+					}
```

Comment:
I'd put this in a try/catch since it can fail, and it does the user won't know what was the actual error